packages/office-ui-fabric-react/src/components/Button/SplitButton/SplitButton.styles.ts

```diff
@@ -14,12 +14,17 @@ export const getStyles = memoizeFunction(
       border: 'none'
     };
 
-    const splitButtonDivider: IStyle = {
+    const splitButtonDividerDisabled: IStyle = {
```

Comment:
You might want to still leave the `splitButtonDivider` styles here, at least the position ones, in order to not affect other components (which appears to be happening in some snapshots).

Reply:
Great catches!